insights/tests/__init__.py

```diff
@@ -56,14 +56,23 @@ def inner(ds, pattern):
 spec_factory.find = _intercept_find(spec_factory.find)
 
 
-def _get_registry_points(component):
+def _get_registry_points(component, filterable_only=False):
```

Comment:
Let's have this function just deal with returning registry points and filter down to just the filterable ones at the point where that's needed.